react/typings/events.d.ts

```diff
@@ -5,6 +5,7 @@ export interface PixelMessage extends MessageEvent {
     | OrderPlacedData
     | OrderPlacedTrackedData
     | PageViewData
+    | LegacyProductViewData
```

Comment:
Fixed a type error in `legacyEvents.ts`: the events were first discriminated by `eventName = 'vtex:pageInfo'` and later `eventType`. We were missing a type with `eventType = 'productView'`.

``` 
modules/legacyEvents.ts:58:14 - error TS2678: Type '"productView"' is not comparable to type '"vtex:promoView" | "vtex:promotionClick" | "homeView" | "productPageInfo" | "internalSiteSearchView" | "categoryView" | "departmentView" | "emptySearchView"'.

58         case 'productView': {
                ~~~~~~~~~~~~~
```